lib/twilio-ruby.rb

```diff
@@ -16,27 +16,20 @@
 require 'twilio-ruby/security/request_validator'
 require 'twilio-ruby/util/configuration'
 
-Dir[File.dirname(__FILE__) + '/twilio-ruby/http/**/*.rb'].sort.each do |file|
+Dir[File.join(__dir__, 'twilio-ruby/compatibility/**/*.rb')].sort.each do |file|
```

Comment:
Also there's no files that match this glob. Maybe it can be removed.

Reply:
Agreed.